src/Pyz/Zed/Stock/StockDependencyProvider.php

```diff
@@ -19,7 +18,7 @@ class StockDependencyProvider extends SprykerStockDependencyProvider
      *
      * @return \Spryker\Zed\Stock\Dependency\Plugin\StockUpdateHandlerPluginInterface[]
      */
-    protected function getStockUpdateHandlerPlugins(Container $container)
+    protected function getStockUpdateHandlerPlugins($container)
```

Comment:
Why that if core has it? https://github.com/spryker/stock/blob/master/src/Spryker/Zed/Stock/StockDependencyProvider.php#L73

Reply:
Stock is still 4.0? https://github.com/spryker-shop/suite/blob/master-merge-availability/composer.lock#L12907